source/shared/cpp/ObjectModel/SubmitAction.cpp

```diff
@@ -63,9 +78,9 @@ std::shared_ptr<BaseActionElement> SubmitActionParser::Deserialize(ParseContext&
     std::shared_ptr<SubmitAction> submitAction = BaseActionElement::Deserialize<SubmitAction>(context, json);
 
     submitAction->SetDataJson(ParseUtil::ExtractJsonValue(json, AdaptiveCardSchemaKey::Data));
+    submitAction->SetConditionallyEnabled(ParseUtil::GetBool(json, AdaptiveCardSchemaKey::ConditionallyEnabled, true));
```

Comment:
Default value for ConditionallyEnabled should be false?

Reply:
yes changed to false.